script/Utils.sol

```diff
@@ -71,13 +72,14 @@ library Utils {
         public
         returns (
             GovernanceERC20,
-            GovernanceERC20.TokenSettings memory,
+            MaciVotingSetup.TokenSettings memory,
             GovernanceERC20.MintSettings memory
         )
     {
-        GovernanceERC20.TokenSettings memory tokenSettings = GovernanceERC20.TokenSettings({
-            name: "DAO Voting Token",
-            symbol: "DVT"
+        MaciVotingSetup.TokenSettings memory tokenSettings = MaciVotingSetup.TokenSettings({
+            addr: address(0), // If set to `address(0)`, a new `GovernanceERC20` token is deployed
+            name: "Test Token",
```

Comment:
lets put `name` and `symbol` as env variables:

```javascript
MaciVotingSetup.TokenSettings memory tokenSettings = MaciVotingSetup.TokenSettings({
          addr: address(0), // If set to `address(0)`, a new `GovernanceERC20` token is deployed
          name: vm.envString("TOKEN_NAME"),
          symbol: vm.envString("TOKEN_SYMBOL")
      });
```


Don't forget to add them in the `.env.example` file 😊